src/interfaces/standings.interface.ts

```diff
@@ -0,0 +1,51 @@
+  import {Problem} from './problem.interface'
+  import {Member} from './member.interface'
+  
+
+  export interface Standings {
+    contest: Contest
+    problems: Problem[]
+    rows: Row[]
+  }
+  
+  export interface Contest {
+    id: number
+    name: string
+    type: string
```

Comment:
Type is an enum with values: `CF, IOI, ICPC`.

I would suggest creating an enum `ContestType` and replace `string` with `ContestType`.